File: TemperatureCon/TemperatureCon.py
# name: TemperatureCon
# description:
# Simple Temperature Converter in Python
#
# author: @TheGoogleRider
# version: 2.1
# date: 06/09/2022

print("Enter '1' for Celsius")
print("Enter '2' for Fahrenheit")
print("Enter '3' for Kelvin")
n = int(input("Enter the unit you want to convert: "))
for _ in range(n):
    if n == 1:
        c = float(input("Input Celsius: "))
        print()
        print("Celsius Input =",c)
        #Celsius to Kelvin
        k = c + 273.15
        print("Kelvin =",k)
        #Celsius to Fahrenheit
        f = c * 9/5 + 32
        print("Fahrenheit =",f)
    elif n == 2:
        f = float(input("Input Fahrenheit: "))
        print()
        print("Fahrenheit Input =",f)
        #Kelvin to Celcius
        c = (f - 32) * 5/9
        print("Celsius = ",c)
        #Fahrenheit to Kelvin
        k = (f - 32) * 5/9 + 273.15
        print("Kelvin = ",k)
    elif n == 3:
        k = float(input("Input Kelvin: "))
        print()
        print("Kelvin Input =",k)
        #Kelvin to Celcius
        c = k - 273.15
        print("Celsius =",c)
        #Kelvin to Fahrenheit
        f = (k - 273.15) * 9/5 + 32
        print("Fahrenheit =",f)
    else:
        print("Please enter an option from the screen")
    exit()
#By TheGoogleRider

File: TemperatureCon/Java/TemperatureCon.java
package TemperatureCon.Java;
/**
 * name: TemperatureCon
 * description:
 * Simple Temperature Converter in Java
 * 
 * author: @TheGoogleRider
 * version: (1.2)
 * date: 12/09/2022
**/

import java.util.*;

public class TemperatureCon{
    public static void main(String[] args){
        System.out.println("TemperatureCon_1.0-Java");
        System.out.println();
        
        try (Scanner sc = new Scanner(System.in)) {
            System.out.println("Enter '1' for Celsius");
            System.out.println("Enter '2' for Fahrenheit");
            System.out.println("Enter '3' for Kelvin");
            System.out.println();
            System.out.print("Enter the unit you want to convert: ");
            int unit = sc.nextInt();
            System.out.println();

                
            if(unit==1){ //Celcius Input
                System.out.print("Input Celcius: ");
                double c = sc.nextDouble();
                System.out.println();
                    
                double f = c * 9/5 + 32;
                System.out.println("Fahrenheit = "+f);
                double k = c + 273.15;
                System.out.println("Kelvin = "+k); 
            }
            
            else if(unit==2){ //Fahrenheit Input
                System.out.print("Input Fahrenheit: ");
                double f = sc.nextDouble();
                System.out.println();

                double c = (f - 32) * 5/9;
                System.out.println("Celcius = "+c);
                double k = (f - 32) * 5/9 + 273.15;
                System.out.println("Kelvin = "+k);  
            }

            else if(unit==3){ //Kelvin Input
                System.out.print("Input Kelvin: ");
                double k = sc.nextDouble();
                System.out.println();

                double c = k - 273.15;
                System.out.println("Celcius = "+c);
                double f = (k - 273.15) * 9/5 + 32;
                System.out.println("Fahrenheit = "+f);
            }
            
            else{
                System.out.println("Please enter a proper Unit");
            }
        }
    }
}

//By TheGoogleRider